packages/block-library/src/search/frontend.js

```diff
@@ -0,0 +1,63 @@
+// eslint-disable-next-line @wordpress/no-global-event-listener
+document.addEventListener( 'DOMContentLoaded', () => {
+	const transitionDuration = 300;
+
+	Array.from(
+		document.getElementsByClassName(
+			'wp-block-search__button-behavior-expand'
+		)
+	).forEach( ( block ) => {
+		const hiddenClass = 'wp-block-search__searchfield-hidden';
+		const wrapperClass = '.wp-block-search__inside-wrapper';
+		const buttonClass = '.wp-block-search__button';
```

Comment:
I think the 3 `const` here can be moved outside the `forEach`loop :+1: 

```suggestion
	const transitionDuration = 300;
	const hiddenClass = 'wp-block-search__searchfield-hidden';
	const wrapperClass = '.wp-block-search__inside-wrapper';
	const buttonClass = '.wp-block-search__button';

	Array.from(
		document.getElementsByClassName(
			'wp-block-search__button-behavior-expand'
		)
	).forEach( ( block ) => {
```